docs/source/cite.rst

```diff
@@ -0,0 +1,10 @@
+.. _cite:
+
+How to cite T3
+===============
+
+Text form::
+
+    A. Grinberg Dana, H. Wu, X. Dong, K.A. Spiekermann, W.H. Green, The RMG-ARC Tandem Tool (T3), version 0.1.0, https://github.com/ReactionMechanismGenerator/T3
```

Comment:
I suggest:
" A. Grinberg Dana, K.A. Spiekermann, X. Dong, H. Wu, W.H. Green, The RMG-ARC Tandem Tool (T3) for automated kinetic model generation and refinement, version 0.1.0, source code: https://github.com/ReactionMechanismGenerator/T3"